addon-test-support/-private/utils.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+export function checkMatcher(message, matcher) {
+  if (typeof matcher === 'string') {
+    return message.includes(matcher);
```

Comment:
I've been stewing on this for 20 minutes now. You make a really good argument about regexp being the "magical" one. I think I'm going to leave it though. More than the risk of breaking addons, I think the ergonomics of partial matching is going to be better in practice.

Always open to revisiting it.